src/seedu/addressbook/data/person/Name.java

```diff
@@ -39,6 +41,10 @@ public static boolean isValidName(String test) {
     /**
      * Retrieves a listing of every word in the name, in order.
      */
+    public List<String> getFindWordsInName() {
```

Comment:
In general, it is not good practice to modify a class because of external reasons that are not related to that class. (e.g. Add method in name instead of find even though it is more relevant to the `FindCommand` class instead of the `Name` class)